docs/2026-06-25_aws_gateway_benchmark/translation/analyze.py

```
@@ -0,0 +1,150 @@
+    for cid, case in caps["cases"].items():
+        bundle = {"case_id": cid, "dialect": case["dialect"], "stream": case["stream"],
+                  "intent_note": case["note"], "client_request": case["client_request"],
+                  "gateways": case["gateways"]}
+        json.dump(bundle, open(os.path.join(d, f"{cid}.json"), "w", encoding="utf-8"), indent=2)
+        ids.append(cid)
+    print(f"wrote {len(ids)} case bundles to {d}")
+    for cid in ids:
+        print("  ", cid)
+
+
+def _cell(items):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_cell`

(ANN202)